docs/specification/STU2.md

```diff
@@ -509,7 +509,7 @@ FHIRcast describes an workflow event subscription and notification scheme toward
 
 New events are proposed in a prescribed format using the [documentation template](../../events/template) by submitting a [pull request](https://github.com/fhircast/docs/tree/master/docs/events). FHIRcast events are versioned, and mature according to the [Event Maturity Model](#event-maturity-model).
 
-FHIRcast events are stateless. Context changes are a complete replacement of any previously communicated context, not "deltas". Understanding an event SHALL not require receiving a previous or future event. 
+FHIRcast events are stateless. For a given event, context elements are a complete replacement of any previously communicated context, not "deltas". Understanding an event SHALL not require receiving a previous or future event. 
```

Comment:
```suggestion
FHIRcast events are stateless. For a given event, context changes are a complete replacement of any previously communicated context, not "deltas". Understanding an event SHALL not require receiving a previous or future event. 
```

Reply:
@wmaethner - do you agree with this small change. Re-reading the comments on https://jira.hl7.org/browse/FHIR-25891, make it seem important.